code-challenges.test.js

```diff
@@ -19,12 +19,34 @@ const numbersArray1 = [6, 7, 8, 9, 10]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+
+
 // b) Create the function that makes the test pass.
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
+// Psuedo Code:
+// first make a test to test the function to see if a number is divisible by three
+// start with a describe statement
+// Describe the divideBYThree function, "itll be a function that returns true if the number is divisible by 3"
+// Next we will use the expect statement to call the function
+// Make three different expect statments 
+// Run the test and watch it fail
+
+
+
 // a) Create a test with expect statements for each of the variables provided.
 
+// describe ("divideByThree", () => {
+//     it("returns true if the mumber is divisible by 3 ", () => {
+//     expect(divideByThree({ 15 })).toBe("15 is divisible by 3")
```

Comment:
Good job constructing your test, but remember to you those variables provided in your expect statements.